### PR TITLE
[cmake] remove usage of kodi-platform

### PR DESCRIPTION
**Pull request by olafhering**
The audiodecoder.2sf binary addon does not use kodi-platform.
Remove existing references from debian/control.

### CODE REVIEW

**Comment by Rechi** [comment]
You have opened the PR against the wrong branch, master branch is no longer used.
Also as long as CI fails, I won't merge a PR, so the kodi-platform build error has to be fixed first.

**Comment by olafhering** [comment]
> You have opened the PR against the wrong branch, master branch is no longer used.

Has this been discussed in public?

**Comment by Rechi** [comment]
If you open https://github.com/xbmc/audiodecoder.2sf, you see it shows Leia (not master) as default branch. Also https://github.com/xbmc/repo-binary-addons/blob/Leia/audiodecoder.2sf/audiodecoder.2sf.txt and https://github.com/xbmc/repo-binary-addons/blob/Matrix/audiodecoder.2sf/audiodecoder.2sf.txt point to Leia.

**Comment by olafhering** [comment]
Yes.

Does that answer the question?

Let me rephrase the question: Was is discussed in public why it is a good idea to proactivly break the commonly used convention "development happens in master branch"? I just want to learn something new. I see xmbc.git#master also switched the addons to a non-master branch.

Why does master still exist if it is not used?

**Comment by Rechi** [comment]
> Why does master still exist if it is not used?

It may is still referenced somewhere and I didn't want to break it.
